### ISSUE TITLE
digitalocean deployment fails with Could not retrieve the list of available versions for provider hashicorp/digitalocean

### ISSUE THREAD

**Issue by brenzi** [w3f/polkadot-validator-setup#147]
I guess the provider should actually be 
https://registry.terraform.io/providers/digitalocean/digitalocean/latest
but how to change that?

```
Initializing provider plugins...

- Finding hashicorp/digitalocean versions matching "~> 1.16"...

- Finding hashicorp/google versions matching "~> 2.15"...

- Installing hashicorp/google v2.20.3...

- Installed hashicorp/google v2.20.3 (signed by HashiCorp)

╷
│ Warning: Version constraints inside provider configuration blocks are deprecated
│
│   on backend.tf line 3, in provider "google":
│    3:   version     = "~>2.15"
│
│ Terraform 0.13 and earlier allowed provider version constraints inside the
│ provider configuration block, but that is now deprecated and will be
│ removed in a future version of Terraform. To silence this warning, move the
│ provider version constraint into the required_providers block.
│
│ (and one more similar warning elsewhere)
╵


╷
│ Error: Failed to query available provider packages
│
│ Could not retrieve the list of available versions for provider
│ hashicorp/digitalocean: provider registry registry.terraform.io does not
│ have a provider named registry.terraform.io/hashicorp/digitalocean
│
│ Did you intend to use digitalocean/digitalocean? If so, you must specify
│ that source address in each module which requires that provider. To see
│ which modules are currently depending on hashicorp/digitalocean, run the
│ following command:
│     terraform providers
╵


Command execution failed with code: 1
(node:2894) UnhandledPromiseRejectionWarning: Error: 1
    at ChildProcess.<anonymous> (/mnt/d/integritee/polkadot-validator-setup/src/lib/cmd.js:45:18)
    at ChildProcess.emit (events.js:376:20)
    at maybeClose (internal/child_process.js:1055:16)
    at Socket.<anonymous> (internal/child_process.js:441:11)
    at Socket.emit (events.js:376:20)
    at Pipe.<anonymous> (net.js:673:12)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:2894) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2894) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 9.95s.
```

**Comment by mmagician** [comment]
There's a PR open for it already: https://github.com/w3f/polkadot-validator-setup/pull/144, which should fix the provider.
Feel free to use [it already](https://github.com/w3f/polkadot-validator-setup/blob/494c8044e34420f092ad31f0a8bfae9f8e3af3ff/terraform/digitalocean/provider.tf) in your setup.
